.circleci/config.yml

```diff
@@ -372,7 +372,7 @@ jobs:
           at: ~/project
       - run:
           name: AcceptanceTests
-          no_output_timeout: 20m
+          no_output_timeout: 30m
```

Comment:
Do we need this increase or was it part of your testing?

Reply:
good catch, I changed the value when I was testing and forgot to roll it back